settings.gradle

```diff
@@ -20,6 +20,8 @@ include 'tests'
 include 'core:python2ActionLoop'
 include 'core:python3ActionLoop'
 include 'core:python3AiActionLoop'
```

Comment:
the runtime Python `3Ai` uses TensorFlow already by the way. It would useful to explain the different between 3TF and 3Ai.

Reply:
OK. I will try to optimize my contribution. Thanks for your reply.

Reply:
Looking forward to your contribution - thank you!